src-built-in/components/appCatalog/src/app.jsx

```diff
@@ -9,47 +9,8 @@ import ReactDOM from "react-dom";
 import "../../../../assets/css/font-finance.css";
 import "../../../../assets/css/finsemble.css";
 import "../appCatalog.css";
-import ComplexMenu from "../../complexMenu/ComplexMenu";
-import AppContent from "./components/AppContent";
-
-
-/**
- * This is our application launcher. It is opened from a button in our sample toolbar, and it handles the launching of finsemble components.
- *
- * @class AppLauncher
- * @extends {React.Component}
- */
-class AppCatalog extends React.Component {
-	constructor(props) {
-		super(props);
-		this.state = {
-			loaded: false,
-			headerImgUrl: ""
-		};
-	}
-	componentWillMount() {
-		FSBL.Clients.ConfigClient.getValues(null, (err, config) => {
-			if (config.startup_app && config.startup_app.applicationIcon) {
-				//console.log("config.startup_app.applicationIcon", config.startup_app.applicationIcon)
-				this.setState({
-					loaded: true,
-					// headerImgUrl: config.startup_app.applicationIcon
-				});
-			}
-		})
-		this.setState({ activeSection: this.props && this.props.activeSection ? this.props.activeSection : '' })//Props did not exist in the constructor
-	}
-	render() {
-		var self = this;
-		if (!this.state.loaded) return null;
-		return (
-			<ComplexMenu headerImgUrl={this.state.headerImgUrl} title="App Catalog" activeSection="Apps" navOptions={[{
-				label: "Apps",
-				content: <AppContent installed={true} />
-			}]} />
-		);
-	}
-}
+import "../../complexMenu/menu.css";
+import { AppCatalog } from "@chartiq/finsemble-ui/src/components/appCatalog/appCatalog";
 
 if (window.FSBL && FSBL.addEventListener) { FSBL.addEventListener("onReady", FSBLReady); } else { window.addEventListener("FSBLReady", FSBLReady) }
```

Comment:
Can we hide the FSBLReady code behind finsemble-ui? This code has no use to the client developers and they probably don't have to see it

Reply:
Agreed! Use the onReady hook. Make one if there isn't one. Then you can just render the component.

Reply:
You can not use hooks outside a component function. Where do you want to use the hook?

It makes no sense to move the following into finsemble-ui
```javascript
	ReactDOM.render(
		<AppCatalog />
		, document.getElementById("bodyHere"));
```
Create another component function? Please make it clear what changes/where you want them to be.

Reply:
I created a component to wrap around the component we're wrapping and only renders it when FSBL is ready: https://github.com/ChartIQ/finsemble-ui/pull/11/files#diff-5d2eca232b4030fea5fff9ca17c73701R126
Eventually we'll refactor this component to a helper function so everyone else can use it